spec/factories/users.rb

```diff
@@ -1,4 +1,6 @@
 FactoryGirl.define do
-  factory :user do
+  factory :user do |f|
+    f.email Faker::Internet.email
```

Comment:
Invoke Faker as a lambda (with curly braces) to avoid having the data in many factories.